libs/blocks/vimeo/vimeo.js

```diff
@@ -1,25 +1,81 @@
 import { createIntersectionObserver, createTag, isInTextNode } from '../../utils/utils.js';
 
-export default function init(a) {
-  if (isInTextNode(a)) return;
-  const embedVimeo = () => {
-    const url = new URL(a.href);
-    let src = url.href;
-    if (url.hostname !== 'player.vimeo.com') {
-      const video = url.pathname.split('/')[1];
-      src = `https://player.vimeo.com/video/${video}?app_id=122963`;
-    }
-    const iframe = createTag('iframe', {
-      src,
-      style: 'border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;',
+function addPrefetch(kind, url, as) {
+  const linkElem = createTag('link', { rel: kind, href: url });
+  if (as) linkElem.as = as;
+  linkElem.crossorigin = true;
+  document.head.append(linkElem);
+}
+
+class LiteVimeo extends HTMLElement {
```

Comment:
Whats the extends `HTMLElement` used for?

Reply:
Is for creating the custom `lite-youtube` and `lite-vimeo` html elements. 
Part of the code is an optimized version of [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) and [lite-vimeo-embed](https://github.com/luwes/lite-vimeo-embed), which are also used in Express.